packages/validate/test/DateValidators.test.js

```diff
@@ -13,7 +13,7 @@ describe('Date Validation', () => {
   it('provides new isDate() to allow only dates', () => {
     let isEnabled;
     const validator = new IsDate();
-    expect(validator.name).to.equal('IsDate');
+    expect(IsDate.validatorName).to.equal('IsDate');
```

Comment:
nit pick 😬 
maybe use `validator.constructor.name`? might be a more realistic use-case 🙈

Reply:
done 😊